src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -18,8 +22,17 @@
      *                 - DAY - текущий день (число месяца);
      *                 В любом другом случае бросить DateTimeException
      **/
+    private static final DateTimeFormatter FORMATTER1 = DateTimeFormatter.ofPattern("yyyyMMdd");
+    private static final DateTimeFormatter FORMATTER2 = DateTimeFormatter.ofPattern("d MMM yyyy");
+    private static final DateTimeFormatter FORMATTER3 =
+            DateTimeFormatter.ofPattern("dd MMMM YYYY HH:mm");
+    private static final int TIME_SHIFT = 2;
+
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        LocalDate now = LocalDate.now();
+        return datePart == DateTimePart.FULL ? String.valueOf(now) : (datePart == DateTimePart.YEAR
+                ? String.valueOf(now.getYear()) : (datePart == DateTimePart.MONTH
+                ? String.valueOf(now.getMonth()) : String.valueOf(now.getDayOfMonth())));
```

Comment:
let's use switch case here, that would be more readable

Reply:
done